### PR TITLE
METPS-233: Implement Job to validate user config files

### PR DESCRIPTION
**Pull request by ynurmahomed**

### CODE REVIEW

File: src/main/resources/pdiresources/transformations/validate-forms.ktr [hunk 1]
```diff
@@ -0,0 +1,914 @@
...
+    <is_key_private>N</is_key_private>
+  </info>
+  <notepads>
+  </notepads>
+  <connection>
+    <name>OpenMRS destination</name>
+    <server>${ETL_DATABASE_HOST}</server>
+    <type>MYSQL</type>
+    <access>Native</access>
+    <database>merge_db</database>
```
Comment (kaweesi) [inline src/main/resources/pdiresources/transformations/validate-forms.ktr]: shouldn't this be ${ETL_SOURCE_DATABASE} instead of 'merge_db'!

Comment (ynurmahomed) [reply src/main/resources/pdiresources/transformations/validate-forms.ktr]: @kaweesi The connection is for the destination DB. We are using approach as in `merge-patient`: https://github.com/esaude/openmrs-database-migrator/blob/9006fc9e1cc9b68ed6bef8bf9d47ba049f2e8cc3/src/main/resources/pdiresources/transformations/merge-patient.ktr#L439-L446

Comment (kaweesi) [reply src/main/resources/pdiresources/transformations/validate-forms.ktr]: is that database different from the source database? if so then we can probably make the name configurable through another property!

**Comment by kaweesi** [comment]
thanks @ynurmahomed for the great work, please also update your passwords to ${ETL_DATABASE_PASSWORD}

File: src/main/resources/pdiresources/transformations/validate-program-workflows.ktr [hunk 2]
```diff
@@ -444,7 +444,7 @@
     <database>merge_db</database>
```
Comment (kaweesi) [inline src/main/resources/pdiresources/transformations/validate-program-workflows.ktr]: made some former suggestion of merge_db being read from a property too.

File: src/main/resources/pdiresources/transformations/validate-forms.ktr (hunk 1 above)
Comment (ynurmahomed) [reply src/main/resources/pdiresources/transformations/validate-forms.ktr]: Done

File: src/main/resources/pdiresources/transformations/validate-program-workflows.ktr (hunk 2 above)
Comment (ynurmahomed) [reply src/main/resources/pdiresources/transformations/validate-program-workflows.ktr]: Done

**Pull request merged**